README.md

```diff
@@ -3,9 +3,20 @@
 The purpose of this project is to provide a generic mechanism to set up a
 gluster-ctdb-samba cluster as a test environment. The mechanism is built on
 vagrant-libvirt and ansible, and should be able to run on any host that has
-support for these pieces of software. It is pulling the latest upstream bits
-from Gluster and Samba (master branches) for setting up the cluster.
+support for these pieces of software. It is pulling the latest nightly RPM
+builds from the upstream master branches of Gluster and Samba for setting
+up the cluster.
 
-In the case of Gluster, the latest nightly RPM builds from master are pulled.
 In the future, we might support options for choosing to consume pre-built RPMS
 or building from a given software branch.
+
+Various branches and resources are playing together to enable this CI:
+
+- [Centos-CI](https://ci.centos.org/) - these are the resource where our tests and builds run.
+- [master branch](https://github.com/gluster/samba-integration/) - this brings up the test environment and then the tests from the tests branch
+- [tests branch](https://github.com/gluster/samba-integration/tree/tests) - the actual test suites to run, invoked by `make test` in the master branch after environment setup (could be a different repo, but we started here for simplicity)
+- [centos-ci branch](https://github.com/gluster/samba-integration/tree/centos-ci) - this contains the script that is run as part of the centos-ci job to prepare the host and trigger our `make test` at the end
+- [samba-build branch](https://github.com/gluster/samba-integration/tree/samba-build) - this contains the specfile and ansible automation to build our nightly samba RPMs
+- https://github.com/gluster/centosci - gluster's centos-ci integration: this contains the centos-ci job definitions for all gluster projects that have one, specifically for the samba-integration, nightly samba builds and nightly gluster builds
```

Comment:
lgtm

nit: just to maintain the uniformity - can we hide centosci repo url too under some tag word like "CentOS-CI jobs" or similar?